### PR TITLE
updated group members / adjusted deploy.py to new account

### PR DESCRIPTION
**Pull request by kremeyer**

### CODE REVIEW

**Comment by LaurentRDC** [comment]
Let me make you an Admin

File: people/students/laurenz.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Laurenz Kremeyer
+position: Ph.D. Student
+email: alice.bob@mail.mcgill.ca
```
Comment (LaurentRDC) [inline people/students/laurenz.md]: Probably not your real e-mail

Comment (kremeyer) [reply people/students/laurenz.md:4]: Oh, thanks for letting me know. Can I change it in this PR, or do I need to open another one?

Comment (LaurentRDC) [reply people/students/laurenz.md]: You can change it in this PR by: 
1. making the change on your computer;
2. committing that change to your local master branch (`git commit -am "your message here"`)
3. pushing. The change should appear here (`git push`)

Comment (kremeyer) [reply people/students/laurenz.md]: Thanks!

**Pull request merged**